lib/active_model/serializer/configuration.rb

```diff
@@ -7,6 +7,7 @@ module Configuration
       included do |base|
         base.config.array_serializer = ActiveModel::Serializer::ArraySerializer
         base.config.adapter = :flatten_json
+        base.config.key_format = nil
```

Comment:
@joaomdmoura I just wanted to be explicit so that anyone spelunking in the code can easily look at this file and see what all the global configuration options are.  I'm not tied to it if ya want me to remove it.